### PR TITLE
feat: update styling

### PR DESCRIPTION
**Pull request by tilmx**
## Proposed Changes
* Update Styling and Linking of Library Connect Teaser
* Prepare Tutorial Teaser
* Update Splash Screen Intro
* Update Tab Bar Styling
* Add Empty State to Property List
* Add Example Download to Splash Screen

### CODE REVIEW

**Comment by marionebl** [comment]
Deployed at: https://alva-issues-716.surge.sh

File: packages/core/src/container/connect-pane-container.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-		`}
-		primaryButton="Connect"
-		secondaryButton="Download Example File with Library "
-		onPrimaryButtonClick={props.onPrimaryButtonClick}
-		onSecondaryButtonClick={props.onSecondaryButtonClick}
-	/>
+	<TeaserRow>
+		<Teaser
+			headline="Connect a Library"
+			description={`
```
Comment (marionebl) [inline packages/core/src/container/connect-pane-container.tsx]:
```suggestion
			description="Use real Code Components for your Prototype"
```

Comment (marionebl) [inline packages/core/src/container/connect-pane-container.tsx]: Simplify to 

```
description="Use real Code Components for your Prototype"
```

File: packages/core/src/container/element-list/element-list.tsx [hunk 2]
```diff
@@ -445,18 +445,15 @@ export class ElementList extends React.Component {
 			>
 				<Components.Element.ElementChildren>
 					{childContent ? <ElementContentContainer content={childContent} /> : null}
-					<div
-						style={{
-							position: 'absolute',
-							opacity: hasChildren ? 0 : 1
-						}}
-					>
+					{hasChildren ? (
```
Comment (marionebl) [inline packages/core/src/container/element-list/element-list.tsx]: Simplify to

```
{!hasChildren && (
  <Components.EmptyState
    headline="Elements"
    copy="Drop Components here from the Library below"
    image={Images.EmptyElements}
    />
)}
```

File: packages/core/src/container/project-settings-container.tsx [hunk 3]
```diff
@@ -17,29 +17,42 @@ export class ProjectSettingsContainer extends React.Component {
...
-							})
-						}
-					>
-						See Library Requirements
-					</Components.Link>
-				</Components.Space>
+				<Components.Headline order={4}>Connected Libraries</Components.Headline>
+				<Components.Space sizeBottom={SpaceSize.S} />
+				<Components.Copy textColor={Color.Grey50}>
+					Connect multiple component libraries to your prototype. Just select the package.json
```
Comment (marionebl) [inline packages/core/src/container/project-settings-container.tsx]: Remove the "just":

`Select the package.json ...`

File: packages/core/src/container/property-list/property-list.tsx [hunk 4]
```diff
@@ -53,7 +53,15 @@ export class PropertyListContainer extends React.Component<PropertyListContainer
 		);
 
 		return (
-			<div>
+			<>
+				{relevantProps.length > 0 ? (
```
Comment (marionebl) [inline packages/core/src/container/property-list/property-list.tsx]: Simplify to 

```
relevantProps.length === 0 && (
  <Components.EmptyState
    headline="Properties"
    copy="Select an Element to edit Properties"
    />
)
```

**Comment by tilmx** [comment]
Good points, thanks!

**Pull request merged**